how-to-docs/drone-how-to.md

```diff
@@ -128,17 +128,16 @@ If your repository is hosted on Gitlab, you don't want to publish your images to
 Register for a free [Quay account](https://quay.io) using your Github account linked to the Home Office organisation.
 
 Once you've logged into Quay check that you have `ukhomeofficedigital` under Users and Organisations.  
-If you do not, [make a request by adding an issue to the Platform Access column](https://github.com/UKHomeOffice/application-container-platform-bau/projects/1)
+If you do not, [submit a request by making an issue on the BAU board](https://github.com/UKHomeOffice/application-container-platform-bau/issues/).
```

Comment:
Should this also be going through the hub?

Reply:
It should really, but right now there isn't a support request for getting added to the organisation, there's only one for getting a quay robot token.

Reply:
Worth adding a Support Request form for this? (/cc @AwkwardBen)

Ideally we'd be able to funnel all user requests through the hub, to keep things simple for the users (and build up a habit of going through the hub).

Reply:
I think so. Since the user will need to create a repo first to do anything with the robot it doesn't make much sense to combine it with the current robot token request form.